packages/react-scripts/config/webpack.config.js

```diff
@@ -278,7 +278,15 @@ module.exports = function(webpackEnv) {
       //   name: false,
       // },
       splitChunks: bpkReactScriptsConfig.enableAutomaticChunking
-        ? { chunks: 'all', name: true }
```

Comment:
FYI this is a backwards incompatible change in terms of chunk names...

Technically if consumers are relying on these names in any way then this change will break it for them.

Reply:
However this will only apply to those codebases who set `enableAutomaticChunking` to true.

Reply:
We've double checked - this is scoped to only 2 codebases 👍